src/Pages/Order/ExpectPayment/ExpectPayment.js

```diff
@@ -0,0 +1,81 @@
+import "./ExpectPayment.scss";
+
+export default class ExpectPayment extends Component {
+  constructor() {
+    super();
+    this.state = {
+      navBtnPushedAt: 0,
+    };
+  }
+
+  clickHandlerChnageNavBtnPushed = (idx) => {
+    this.setState({
+      navBtnPushedAt: idx,
+    });
+  };
+
+  render() {
+    const { clickHandlerChangeExpectPaymentClickState } = this.props;
+    const { navBtnPushedAt } = this.state;
+    return (
+      <div className="ExpectPayment">
+        <div
+          className="exitModal"
+          onClick={clickHandlerChangeExpectPaymentClickState}
+        />
+        <div className="wrapmodalContainer">
+          <div className="modalLeftNav">
+            <div className="wrapModalLogo">
+              <img
+                className="modalLogo"
+                alt="modalLogo"
+                src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHByZXNlcnZlQXNwZWN0UmF0aW89InhNaWRZTWlkIE1lZXQiIHdpZHRoPSIxMDAiIGhlaWdodD0iMTAwIj48cGF0aCBjbGFzcz0idGRzLWljb24tZmlsbC0tcHJpbWFyeSIgZD0iTTUwIDk5LjhsMTQtNzguN2MxMy4zIDAgMTcuNSAxLjUgMTguMSA3LjQgMCAwIDguOS0zLjMgMTMuNS0xMC4xQzc4IDEwLjMgNjAuMyA5LjkgNjAuMyA5LjlMNTAgMjIuNSAzOS43IDkuOXMtMTcuNy40LTM1LjMgOC41YzQuNSA2LjggMTMuNSAxMC4xIDEzLjUgMTAuMS42LTYgNC44LTcuNCAxOC4xLTcuNGwxNCA3OC43eiIvPjxwYXRoIGNsYXNzPSJ0ZHMtaWNvbi1maWxsLS1wcmltYXJ5IiBkPSJNNTAgNi4zYzE0LjItLjEgMzAuNSAyLjIgNDcuMiA5LjUgMi4yLTQgMi44LTUuOCAyLjgtNS44QzgxLjggMi43IDY0LjcuMyA1MCAuMiAzNS4zLjMgMTguMiAyLjcgMCAxMGMwIDAgLjggMi4yIDIuOCA1LjggMTYuNy03LjMgMzMtOS42IDQ3LjItOS41eiIvPjwvc3ZnPg=="
+              />
+            </div>
+            <div className="modalNavContainer">
+              <ul className="modalNavList">
+                {navTitle.map((title, idx) => {
+                  return (
+                    <li
+                      className={
+                        navBtnPushedAt === idx
+                          ? "modalNavTitle pushed"
+                          : "modalNavTitle normal"
+                      }
+                      key={idx}
+                      onClick={() => this.clickHandlerChnageNavBtnPushed(idx)}
+                    >
+                      {title}
+                    </li>
+                  );
+                })}
+                <span
+                  className={
+                    navBtnPushedAt === 0
+                      ? "modalLeftNavActiveLine first"
+                      : navBtnPushedAt === 1
+                      ? "modalLeftNavActiveLine second"
+                      : "modalLeftNavActiveLine third"
+                  }
+                />
+              </ul>
+            </div>
+          </div>
+          <div className="modalContent">
+            {(() => {
+              if (navBtnPushedAt === 0) return <FinancingOption />;
+              if (navBtnPushedAt === 1) return <FuelCostReduction />;
+              if (navBtnPushedAt === 2) return <IncentiveSystem />;
+            })()}
```

Comment:
```suggestion
            {navBtnPushedAt === 0 && <FinancingOption />}
            {navBtnPushedAt === 1 && <FuelCostReduction />}
            {navBtnPushedAt === 2 && <IncentiveSystem />}
```
굳이 즉시실행 함수로 만들어서 if문을 쓸 필요가 없어보입니다

Reply:
오우 수정완료요!

Reply:
```suggestion
            const ActiveModalCompo = [ <FinancingOption />, <FuelCostReduction />,  <IncentiveSystem />];
            
            {ActiveModalCompo[navBtnPushedAt]}
```

이렇게 바꿨는데 어때요?

Reply:
컴포넌트라서 첫글자는 대문자로 했어요